docs/additional-functionality/gds-spilling.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: GPUDirect Storage (GDS) Spilling
+parent: Additional Functionality
+nav_order: 6
+---
+# GPUDirect Storage (GDS) Spilling
+---
+**NOTE**
+
+_GPUDirect Storage (GDS) Spilling_ is a beta feature!
+
+---
+The [RAPIDS Shuffle Manager](rapids-shuffle.md) has a spillable cache that keeps GPU data in device
+memory, but can spill to host memory and then to disk when the GPU is out of memory. Using
+[GPUDirect Storage (GDS)](https://docs.nvidia.com/gpudirect-storage/), device buffers can be spilled
+directly to storage. This direct path increases system bandwidth, decreases latency and 
+utilization load on the CPU.
+
+### System Setup
+In order to enable GDS spilling, GDS must be installed on the host. GDS software can be 
+downloaded [here](https://developer.nvidia.com/gpudirect-storage). Follow the
+[GDS Installation and Troubleshooting Guide](
+https://docs.nvidia.com/gpudirect-storage/troubleshooting-guide/index.html)
+to install and configure GDS.
+
+### Spark App Configuration
+After GDS is installed on the host, to enable GDS spilling:
+* Make sure the RAPIDS Shuffle Manager is enabled and configured correctly.
+* Make sure the Spark "scratch" directory configured by `spark.local.dir` supports GDS.
+* Add the following configuration to the Spark app:
+```shell
+...
+--conf spark.rapids.memory.gpu.direct.storage.spill.enabled=true" \
```

Comment:
There's a dangling quote here.  I personally would remove the `...` and backslash and just list the single `--conf` line, or possibly only mention the conf key and value.